library/Zend/Http/Response.php

```diff
@@ -250,6 +249,17 @@ public function setStatusCode($code)
     }
 
     /**
+     * Check is correct status code
+     *
+     * @param int $code
+     * @return bool
+     */
+    public static function isStatusCode($code)
+    {
+        return is_numeric($code) && defined('static::STATUS_CODE_' . $code);
```

Comment:
I don't know if that's the best way to check if a status code is valid, because I can create my own status code `555` for a response, and I can't know if is a status code in this way.